cvmfs/cvmfs_server

```diff
@@ -1939,6 +1939,7 @@ CVMFS_HASH_ALGORITHM=$hash_algo
 CVMFS_AUTO_TAG=$autotagging
 CVMFS_GARBAGE_COLLECTION=$garbage_collectable
 CVMFS_AUTO_REPAIR_MOUNTPOINT=true
+CVMFS_AUTOCATALOG=false
```

Comment:
Probably better `CVMFS_AUTOCATALOGS` (plural) so that it is clear it's not just one.